cli/ceph/config.py

```diff
@@ -0,0 +1,19 @@
+from cli import Cli
+
+
+class Config(Cli):
+    """This module provides CLI interface to manage the balancer module."""
+
+    def __init__(self, nodes, base_cmd):
+        super(Config, self).__init__(nodes)
+
+        self.base_cmd = f"{base_cmd} config"
+
+    def set(self, key, value):
+        """set config key to value
+        Args:
+            key (str): config key
+            value (str): config key value
+        """
+        cmd = f"{self.base_cmd} set {key} {value}"
+        return self.execute(sudo=True, check_ec=False, long_running=True, cmd=cmd)
```

Comment:
Can we remove "check_ec=False, long_running=False" since by defult it is false and get parmeter from kw

https://github.com/red-hat-storage/cephci/blob/master/cli/__init__.py#L5

Example:https://github.com/red-hat-storage/cephci/blob/master/cli/ceph/mgr.py#L27